homework.md

```diff
@@ -4,38 +4,38 @@
 
 What is 2 + 2?
 
-5
```

Comment:
Hey Douglas, well done for answering the questions.

Reply:
Thank you Chandre.